recipes/vtk-cdat/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = "2.10" %}
+{% set name = "vtk-cdat" %}
+
+package:
+    name: {{ name }}
+    version: 7.1.0.{{ version }}
+
+source:
+    git_rev: 1c03d1d03b1c12d99dd090ba4c5ca078e2d9612f
+    git_url: git://github.com/Kitware/VTK.git
```

Comment:
@sankhesh we can use our fork. @isuruf as Sankhesh explained we are using VTK and are usually adding filters or fixing VTK bug so our stable releases are based on a stable VTK SHA1 but not an official. Our nightlies (in between version) will not be hosted on conda-forge but on our conda channel (`uvcdat`). I will clean all this up as soon as we have a passing recipe, right now the focus is on getting it to pass travis.